spacy/cli/download.py

```diff
@@ -63,6 +64,16 @@ def download(
         compatibility = get_compatibility()
         version = get_version(model_name, compatibility)
 
+    # If we already have this version installed, skip downloading
```

Comment:
I think it might make sense to get the version from `importlib` instead?

Reply:
Good point, that ends up simpler.

Reply:
Sorry for editing the comment, a new comment would have been clearer. I think you can just use `get_package_version` directly?

Reply:
Ah, I see what you mean - done.